recipe/meta.yaml

```diff
@@ -1,25 +1,26 @@
 {% set name = "conda-index" %}
-{% set version = "0.4.0" %}
+{% set version = "0.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 0d83fd01af3c5b17860a86c1e217090a19b6e262114a242bc7f4ff212ed1e705
+  sha256: 12d78ec6dc55f64ad68761108a416ebde3dc6472f257045ddfa45378980c9074
 
 build:
   number: 0
-  skip: True  # [py<37]
-  script: {{ PYTHON }} -m pip install --no-build-isolation --no-deps . -vv
-  noarch: python
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - flit-core
     - python
     - pip
+    - wheel
+    - setuptools
```

Comment:
```suggestion
```
`flit-core` is used instead of setuptools, so you don't need to add `setuptools` and `wheel`. See https://github.com/conda/conda-index/blob/main/pyproject.toml#L2.